parts/k8s/cloud-init/artifacts/cis.sh

```diff
@@ -87,7 +87,6 @@ assignFilePermissions() {
     chmod 644 /etc/modprobe.d/CIS.conf || exit $ERR_CIS_ASSIGN_FILE_PERMISSION
     chmod 644 /etc/ssh/sshd_config || exit $ERR_CIS_ASSIGN_FILE_PERMISSION
     chmod 600 /etc/security/pwquality.conf || exit $ERR_CIS_ASSIGN_FILE_PERMISSION
-    chmod 400 /boot/grub/grub.cfg || exit $ERR_CIS_ASSIGN_FILE_PERMISSION
```

Comment:
no point in setting this permission if it is getting overwritten by something out of our control